src/Analysis/GWASWizard/GWASQuantitative.jsx

```diff
@@ -132,7 +132,16 @@ const GWASQuantitative = ({ resetGWASType, refreshWorkflows }) => {
               <h4>&nbsp;Tutorial</h4>
             </div>
             <Space direction={'vertical'} align={'center'} style={{ width: '100%' }}>
-              <h4 className='GWASUI-selectInstruction' data-tour='quant-step-2-choosing-variable'>In this step, you will select the harmonized variables for your study. Please select all variables you wish to use in your model, including both covariates and phenotype. (Note: population PCs are not included in this step)</h4>
+              <h4 className='GWASUI-selectInstruction' data-tour='quant-step-2-choosing-variable'>
+                In this step, you will select the harmonized variables for your study.
```

Comment:
I've always wondered since the instructions are so similar between case control and quan for these step descriptions if we should have it just be a constant.